tests/simulation/test_simulate.py

```diff
@@ -40,7 +40,7 @@ def simulate_inputs():
     model_config = get_model_config("iskhakov_et_al_2017_stripped_down", n_periods=1)
     model = process_model(model_config)
 
-    _, space_info = create_state_choice_space(
+    _, sc_space_info = create_state_choice_space(
```

Comment:
```suggestion
    sc_space_info = create_state_choice_space( ... )[1]
```

Won't flag it anymore, if you agree, maybe use a regex to search for that pattern.